Cubical/Foundations/Groupoid.agda

```diff
@@ -0,0 +1,100 @@
+{-
+
+Proof of some groupoid laws for equality
```

Comment:
How about calling this file "GroupoidLaws"? We should make an issue for proving the rest of the laws (this should be a good exercise for a newcomer)

Reply:
Agreed. I think that Bruno is currently working on them, by the way.

Reply:
Yeah, let's synch with him tomorrow at the Agda meeting  (@bbentzen)

Reply:
Yeah, let's do that, I have almost all the laws, except for associativity (I hope to prove it tomorrow).